src/Tribe/Service_Providers/Tooltip.php

```diff
@@ -16,34 +19,25 @@ class Tooltip extends \tad_DI52_ServiceProvider {
 	 * @since 4.9.8
 	 */
 	public function register() {
-		tribe_singleton( 'tooltip.view', '\Tribe\Tooltip\View' );
+		tribe_singleton( 'tooltip.view', View::class );
 
-		$this->hook();
+		$this->register_assets();
```

Comment:
Just to confirm, it's intentional to also not run this on the tribe_common_loaded action, correct?